prep-javascript-operators-and-expressions/main.js

```diff
@@ -0,0 +1,34 @@
+const number1 = 1;
+const number2 = 2;
+const product = number1 * number2;
+console.log('the value of product: ', product);
+console.log('the type of the product: ', typeof product);
+
+const charge = 2;
+const payment = 1;
+const amountRemaining = charge * payment;
```

Comment:
You should be using subtraction here! You should be subtracting `payment` from `charge`

Reply:
ops.. missed that